File: Week 1/cash/cash.c
#include <stdio.h>
#include "cs50.c"
#include <math.h>

int main(void)
{
    // Initializing variables;
    float amount;
    int coins = 0;

    // User input
    do
    {
        amount = get_float("Change amount: ");
    }
    while (amount < 0);

    // Convert Dollars to cents
    amount = round(amount * 100);

    // Check number of quaters(25)
    while (amount > 24)
    {
        amount = amount - 25;
        coins++;
    }
    
    // Check number of dimes(10)
    while (amount > 9)
    {
        amount = amount - 10;
        coins++;
    }

    // Check number of nickels(5)
    while (amount > 4)
    {
        amount = amount - 5;
        coins++;
    }

    // Check number of pennies
    coins = amount + coins;

    printf("%i\n", coins);
}

File: Week 6/Cash/cash.py
amount = -1
coins = 0

while (amount < 0):
    try:
        amount = float(input("Change owed: "))
    except ValueError:
        pass

# Dollars to cents
amount = round(amount * 100)

# Check number of quaters(25)
while (amount > 24):
    amount = amount - 25
    coins += 1

# Check number of dimes(10)
while (amount > 9):
    amount = amount - 10
    coins += 1


# Check number of nickels(5)
while (amount > 4):
    amount = amount - 5
    coins += 1

print(coins + amount)